src/Components/NavBar/NavItem.tsx

```diff
@@ -59,6 +61,14 @@ export const NavItem: React.FC<NavItemProps> = ({
     }
   }
 
+  const trackHover = () => {
+    if (isString(children))
```

Comment:
Can we add a comment above this line mentioning that `children` refers text that's passed into `<NavItem>`.